source/shared/cpp/ObjectModel/ShowCardAction.cpp

```diff
@@ -45,7 +45,7 @@ std::shared_ptr<BaseActionElement> ShowCardActionParser::Deserialize(
     std::shared_ptr<ShowCardAction> showCardAction = BaseActionElement::Deserialize<ShowCardAction>(json);
 
     std::string propertyName = AdaptiveCardSchemaKeyToString(AdaptiveCardSchemaKey::Card);
-    showCardAction->SetCard(AdaptiveCard::Deserialize(json.get(propertyName, Json::Value()), elementParserRegistration, actionParserRegistration));
+    showCardAction->SetCard(AdaptiveCard::Deserialize(json.get(propertyName, Json::Value()), std::numeric_limits<double>::max(), elementParserRegistration, actionParserRegistration));
```

Comment:
just curious, why max?

Reply:
Per discussion with Matt, show cards don't have a separate version, so we don't need to enforce version here. Pass max to effectively say we can handle anything. (because we've already checked version at the top of the main card)

---
In reply to: [172352922](https://github.com/Microsoft/AdaptiveCards/pull/1216#discussion_r172352922) [](ancestors = 172352922)